src/Tools/CoreFx.Tools/project.json

```diff
@@ -20,10 +21,12 @@
         "portable-net45+win8+wpa81"
       ]
     },
-    "net45": {
+    "net46": {
       "dependencies": {
-        "Microsoft.TargetingPack.NETFramework.v4.5": "1.0.1"
+        "Microsoft.TargetingPack.NETFramework.v4.6": "1.0.1",
+        "System.Reflection.Metadata": "1.1.0",
+        "System.Collections.Immutable": "1.1.37"
```

Comment:
Think this was another change because of the desktop build, will check if I can upgrade